src/StatsBase.jl

```diff
@@ -156,7 +156,8 @@ module StatsBase
     residuals,
     r2,
     r²,
-    model_response
+    model_response,
+    ConvergenceException
```

Comment:
Happy to do it : ) Would that be because `ConvergenceException` forms a new "grouping" on its own?